### PR TITLE
GROWTH-738 added robots meta option to default template, added noindex/nof…

### PR DESCRIPTION
**Pull request by cannedyeti**
…ollow to duplicate pages.

### CODE REVIEW

File: _pages/algorithm-development/advanced-algorithm-development/local-development.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ show_related: true
 author: jon_peck
 image:
     teaser: /post_images/local_development/local_development.png
+robots: noindex/nofollow
```
Comment (eliperelman) [inline _pages/algorithm-development/advanced-algorithm-development/local-development.md]: The syntax [according to Google](https://developers.google.com/search/reference/robots_meta_tag) is specified as comma-separated values, so these would be `noindex, nofollow`. Preferably, these can also be combined into [a single directive](https://developers.google.com/search/reference/robots_meta_tag#directives_1) of `none` to achieve the same goal:

```md
robots: none
```

**Review by eliperelman — CHANGES_REQUESTED** [review]
Looks good, see review comment about use of `none`.

File: _pages/algorithm-development/advanced-algorithm-development/local-development.md (hunk 1 above)
Comment (cannedyeti) [reply _pages/algorithm-development/advanced-algorithm-development/local-development.md]: Ahh you're right. Thanks for the heads up. Should be updated now.

File: _pages/platform.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ redirect_from:
  - /basics/
  - /application-development/basic-guides/
  - /basics/algorithm_basics/
+robots: noindex/nofollow
```
Comment (eliperelman) [inline _pages/platform.md]: Have one more instance here.

**Review by eliperelman — APPROVED** [review]
Looks good! Just one more place to change to `none`.

**Comment by bnlb** [comment]
@cannedyeti merging this so we can ship with Tuesday's deploy.

**Pull request merged**